components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3427,12 +3441,12 @@ subroutine cloud_sedimentation(kts,kte,ktop,kbot,kdir,   &
                qc_notsmall_c2: if (qc_incld(k)>qsmall) then
                   !-- compute Vq, Vn
                   call get_cloud_dsd2(qc_incld(k),nc_incld(k),mu_c(k),rho(k),nu,dnu,   &
-                  lamc(k),tmp1,tmp2,lcldm(k))
+                       lamc(k),tmp1,tmp2,lcldm(k))
 
                   nc(k) = nc_incld(k)*lcldm(k)
-                  dum = 1._rtype/lamc(k)**bcn
-                  V_qc(k) = acn(k)*gamma(4._rtype+bcn+mu_c(k))*dum/(gamma(mu_c(k)+4._rtype))
-                  V_nc(k) = acn(k)*gamma(1._rtype+bcn+mu_c(k))*dum/(gamma(mu_c(k)+1._rtype))
+                  dum = 1._rtype / (lamc(k)*lamc(k))
```

Comment:
We should discuss this change with @AaronDonahue. bcn is a constant that is used in many spots. It's possible it doesn't have to be 2. This change would mean this line of code wouldn't change in sync with all the other bcn-dependent computations if bcn's value were changed. One possible solution: For both bcn and eri, define bcn_int, eri_int and use these for exponents. Then if bcn is made to be 2.5, day, bcn_int's def could be removed and thereby reveal where an integer assumption had been made, which could then be reverted. On the C++ side, you could make `template <int e, typename T> intpow(const T& base)` and use it as `intpow<c::bcn_int>(lamc(k))`.

Reply:
@ambrad, yes good catch.  I don't forsee `bcn` changing anytime soon but I think it is safe to assume that it was made into a named constant for just the reason that it could be changed.